pkg/vpcmodel/grouping.go

```diff
@@ -575,3 +564,62 @@ func (details *srcDstDetails) explanationEncode(c *VPCConfig) string {
 	}
 	return connStr + routingStr + egressStr + ingressStr
 }
+
+// cacheGroupedElements functionality
+// ///////////////////////////////////////////////////////
+// 1. functionality related to cachedGrouped.groupedEndpointsElemsMap
+// gets pointer of an element semantically equiv to grouped in cachedGrouped.groupedEndpointsElemsMap
+// if exists, nil otherwise
+func (cachedGrouped *cacheGroupedElements) getExistEndpointElemFromCache(
+	grouped groupedEndpointsElems) *groupedEndpointsElems {
+	// since the endpoints (vsis/subnets) are sorted before printed, grouped.Name() will be identical
+	// to equiv groupedEndpointsElems
+	if existingGrouped, ok := cachedGrouped.groupedEndpointsElemsMap[grouped.Name()]; ok {
+		return existingGrouped
+	}
+	return nil
+}
+
+// gets pointer of an element semantically equiv to grouped in cachedGrouped.groupedEndpointsElemsMap
```

Comment:
move this part with all methods for `cacheGroupedElements` to `unifyGrouping.go` where this type is defined

Reply:
did you push this commit? I don't see any change

Reply:
oops. Pushed now